1.data-exploration/scripts/nbconverted/test-train_split.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import sys
+import pathlib
+import numpy as np
+import pandas as pd
+sys.path.insert(0, "./0.data-download/scripts/")
+from data_loader import load_data
+import matplotlib.pyplot as plt
+from matplotlib.pyplot import figure
+from sklearn.decomposition import PCA
+from tensorflow import keras
+
+from tensorflow.keras.models import Model, Sequential
+import seaborn
+import random as python_random
+import tensorflow as tf
```

Comment:
Please use only minimal package imports - only import code you need. (e.g. PCA isn't used, so please remove import)